### PR TITLE
fix jsonpath usages

### PR DESCRIPTION
**Pull request by atiratree**
to fix incorrect usages of jsonpath (https://github.com/openshift/must-gather/pull/204/files#r683645122)

### CODE REVIEW

File: collection-scripts/gather_priority_and_fairness [hunk 1]
```diff
@@ -38,11 +38,9 @@ fi
...
 # Fetches the POD_NAME where app label is apiserver.
 # Example:
-#  oc get po -n openshift-kube-apiserver -l apiserver -o jsonpath='{range .items[*]}{.metadata.name}{"\n"}'
-#  kube-apiserver-master-0.example.com
-#  kube-apiserver-master-1.example.com
-#  kube-apiserver-master-2.example.com
-KUBE_API_SERVERS=$(oc get po -n openshift-kube-apiserver -l apiserver -o jsonpath='{range .items[*]}{.metadata.name}{"\n"}')
+#  oc get po -n openshift-kube-apiserver -l apiserver -o jsonpath='{.items[*].metadata.name}'
+#  kube-apiserver-master-0.example.com kube-apiserver-master-1.example.com kube-apiserver-master-2.example.com
+KUBE_API_SERVERS=$(oc get po -n openshift-kube-apiserver -l apiserver -o jsonpath='{.items[*].metadata.name}')
```
Comment (atiratree) [inline collection-scripts/gather_priority_and_fairness:43]: no need for range unless we want to print them nicely somewhere

File: collection-scripts/gather_core_dumps [hunk 2]
```diff
@@ -42,7 +42,7 @@ if [ $# -eq 0 ]; then
 fi
 
 PIDS=()
-NODES="${*:-$(oc get nodes -o jsonpath='{range .items[*]}{@.metadata.name} {.status.nodeInfo.operatingSystem==linux}')}"
+NODES="${*:-$(oc get nodes -o jsonpath='{.items[?(@.status.nodeInfo.operatingSystem=="linux")].metadata.name}')}"
```
Comment (atiratree) [inline collection-scripts/gather_core_dumps:45]: filtering wasn't working properly before

**Review by soltysh — APPROVED** [review]
/lgtm
/approve

**Pull request merged**